projects/plugins/social/src/js/components/social-notes-toggle/index.tsx

```diff
@@ -58,6 +69,56 @@ const SocialNotesToggle: React.FC< SocialNotesToggleProps > = ( { disabled } ) =
 					'jetpack-social'
 				) }
 			</Text>
+			{ isEnabled && ! isUpdating ? (
+				<div className={ styles[ 'notes-options-wrapper' ] }>
+					<ToggleControl
+						label={ __( 'Append post link', 'jetpack-social' ) }
+						checked={ notesConfig.append_link ?? true }
+						className={ styles.toggle }
+						onChange={ onToggleNotesConfig( 'append_link' ) }
+						help={ __(
+							'Whether to append the post link when sharing the note.',
+							'jetpack-social'
+						) }
+					/>
+					{ notesConfig.append_link ? (
+						<>
+							<ToggleControl
+								label={ __( 'Use shortlink', 'jetpack-social' ) }
+								checked={ notesConfig.use_shortlink ?? false }
+								className={ styles.toggle }
+								onChange={ onToggleNotesConfig( 'use_shortlink' ) }
+								help={ __(
+									'Whether to use the shortlink instead of the full URL.',
+									'jetpack-social'
+								) }
+							/>
+							<ToggleControl
+								label={ __( 'Use parenthical link format', 'jetpack-social' ) }
```

Comment:
Nitpick: Should we simply say `Use permashortcitation`? When I google parenthical link format, I don't get any meaningful results. If I google permashortcitation, the first result is that of the indieweb documentation stub article.

Reply:
Yeah, that's better if it helps users.

Reply:
Updated ✅